.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main, b0.69, b0.70 ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '39 19 * * 5'
```

Comment:
Every Friday at 3:39 pm EDT (Saturday at 1:09 am IST)?  That seems like an odd time.

I suppose that Friday/Saturday is good, but it seems like we should wait until after the work day is over.

Reply:
All this was auto-generated by "pressing the button" that generated this PR.

Reply:
What time was it when you pressed it?  😆